frontend/components/Header/Nav/Links/index.js

```diff
@@ -1,18 +1,99 @@
 // @flow
-import styled from 'react-emotion';
+import { css, cx } from 'react-emotion';
 import { connect } from 'unistore/react';
+import palette from '@guardian/pasteup/palette';
+import { textSans } from '@guardian/pasteup/fonts';
+import { tablet, desktop } from '@guardian/pasteup/breakpoints';
+import type { LinkType } from '../__config__';
 
 import SupportTheGuardian from './SupportTheGuardian';
-import Link from './Link';
-import Search from './Search';
 
-import type { LinkType } from '../__config__';
+const link = css`
+    font-size: 14px;
+    font-family: ${textSans};
+    color: ${palette.neutral['1']};
```

Comment:
Can we use the numeric value for the index, as in `palette.neutral[1]`?

Weird that Prettier or ESLint hasn't picked this up 😕

Reply:
We could, but it's defined using the string form- and while this does work. I don't feel 100% happy with it.

P.S. @zeftilldeath says that the palette has different names now, so we'd probably make that change.

Reply:
Cool, we'll tackle this when we tackle all the colours then